### PR TITLE
All sample inputs as files instead of strings [VS-1019]

### PR DESCRIPTION
**Pull request by rsasch**
Ran into what looks like a limit of the WDL `read_string()` function, so converted all the sample-based inputs to be files (instead of strings).

Run that worked (until quotas kicked in): https://app.terra.bio/#workspaces/gvs-dev/GVS_190k_Exomes/job_history/49b92660-e5d6-4305-9857-1dcd19dde1e7

### CODE REVIEW

**Review by mcovarr — APPROVED** [review]

File: scripts/variantstore/wdl/GvsBulkIngestGenomes.wdl [hunk 1]
```diff
@@ -239,10 +241,11 @@ task SplitBulkImportFofn {
         cut -f 1 ~{import_fofn} > sample_names.txt
         cut -f 2 ~{import_fofn} > vcf_file_names.txt
         cut -f 3 ~{import_fofn} > vcf_index_file_names.txt
+        wc -l < ~{import_fofn} > sample_num.txt
     >>>
 
     runtime {
-        docker: "us.gcr.io/broad-dsde-methods/variantstore:2023-07-24-alpine-bd6b9d62e"
+        docker: "gcr.io/google.com/cloudsdktool/cloud-sdk:426.0.0-alpine"
```
[the remark this answers is not in the document]
Comment (rsasch) [reply scripts/variantstore/wdl/GvsBulkIngestGenomes.wdl:248]: Happy to replace it with something else, but this looked like the smallest image that would do the trick and we are already using.  Any suggestions?

[the remark this answers is not in the document]
Comment (mcovarr) [reply scripts/variantstore/wdl/GvsBulkIngestGenomes.wdl:248]: This is fine for now, at some point we should do a pass of upgrading everything but that certainly doesn't have to be in this PR

File: scripts/variantstore/wdl/GvsAssignIds.wdl [hunk 2]
```diff
@@ -133,8 +136,7 @@ task AssignIds {
     set -o errexit -o nounset -o pipefail -o xtrace
 
     # make sure that sample names were actually passed, fail if empty
-    num_samples=~{length(sample_names)}
-    if [ $num_samples -eq 0 ]; then
+    if [ ~{num_samples} -eq 0 ]; then
       echo "No sample names passed. Exiting"
```
Comment (gbggrant) [inline scripts/variantstore/wdl/GvsAssignIds.wdl:140]: nit: On first read this sounds like they 'passed', like succe
```suggestion
      echo "No sample names found in file ~{sample_names}. Exiting"
```

**Review by gbggrant — APPROVED** [review]
Looks good.

File: scripts/variantstore/wdl/GvsBulkIngestGenomes.wdl (hunk 1 above)
[the remark this answers is not in the document]
Comment (rsasch) [reply scripts/variantstore/wdl/GvsBulkIngestGenomes.wdl:248]: slim is 410.5 MB, alpine is 233.4 MB

[the remark this answers is not in the document]
Comment (mcovarr) [reply scripts/variantstore/wdl/GvsBulkIngestGenomes.wdl:248]: Yeah alpine is preferable if it works for the task at hand

**Review by koncheto-broad — CHANGES_REQUESTED** [review]
Note: if the changes make the original WDLs unrunnable through normal means, you'll need to document that somewhere.  Up until now, we are still able to run these individual steps to develop on them, debug them, or even push a custom callset through piecemeal.  If this changes our ability to do that, we'll need to document it (or create a ticket to do so).

This is the same feedback given to the PR that joined the output from the early stages of bulk ingest into a single, multi-columned csv.  In that one, Miguel chose to implement it in such a way as to leave the others runnable.

**Review by koncheto-broad — APPROVED** [review]
Thanks for updating the instructions!  I'd prefer if we didn't have to change the inputs this way at all, but this is the most direct way to get around the read_lines() bug at scale

**Pull request merged**